GrandCentralBoard/Widgets/Harvest/HarvestWidgetSettings.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  HarvestWidgetSettings.swift
+//  GrandCentralBoard
+//
+//  Created by Karol Kozub on 2016-04-11.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Decodable
+
+
+struct HarvestWidgetSettings : Decodable {
+    let accessToken: String
+    let refreshToken: String
+    let expirationTime: NSTimeInterval
+    let refreshInterval: NSTimeInterval
+
+    static func decode(json: AnyObject) throws -> HarvestWidgetSettings {
+        let accessToken: String = try json => "accessToken"
+        let refreshToken: String = try json => "refreshToken"
+        let expirationTime: Double = try json => "expirationTime"
+        let refreshInterval: Double = try json => "refreshInterval"
+
+        return HarvestWidgetSettings(accessToken: accessToken, refreshToken: refreshToken, expirationTime: expirationTime, refreshInterval: refreshInterval)
```

Comment:
I think you could have these lines in one statement `return try HarvestWidgetSettings(accessToken: json => "accessToken", ...`